### PR TITLE
chore: Tweak email templates contents

### PR DESCRIPTION
**Pull request by ashindey12**
# Pull request checklist

**Detail as per issue below (required):**

fixes: #9027 Updated email template texts

### CODE REVIEW

File: packages/cognito-custom-mail-lambda/src/mailer/templates/admin-user-invite.mjml [hunk 1]
```diff
@@ -30,11 +30,11 @@
...
+        <mj-text mj-class="body-text">Please see below your userName and temporary password. When you click the login link, you will be re-directed to a screen where you will be asked to change your temporary password.</mj-text>
         <mj-text mj-class="body-text"><strong>User Name: {username}</strong></mj-text>
-        <mj-text mj-class="body-text"><strong>Password: {####}</strong></mj-text>
+        <mj-text mj-class="body-text"><strong>Temporary Password: {####}</strong></mj-text>
         <mj-button href={url}>login</mj-button>
-        <mj-text mj-class="body-text">When you have successfully changed your password, you will then be presented with a list of products and services available to you.</mj-text>
+        <mj-text mj-class="body-text">When you have successfully changed your temporary password, you will then be presented with a list of products and services available to you.</mj-text>
         <mj-text mj-class="body-text">Please do not reply to this email as this mailbox is not monitored. If you are a Reapit Customer and you are having an issue or if you have a question about Reapit Connect, please visit the Reapit Service Desk Portal.</mj-text>
```
Comment (plittlewood-rpt) [inline packages/cognito-custom-mail-lambda/src/mailer/templates/admin-user-invite.mjml:38]: Missing the update about the service desk

File: packages/cognito-custom-mail-lambda/src/mailer/templates/admin-user-invite.mjml [hunk 2]
```diff
@@ -30,11 +30,11 @@
         <mj-text mj-class="body-text">Hi {name},</mj-text>
         <mj-text mj-class="body-text">Welcome to Reapit Connect.</mj-text>
         <mj-text mj-class="body-text">Reapit Connect is our single sign on solution which allows you to seamlessly access products and services provided by Reapit Ltd.</mj-text>
-        <mj-text mj-class="body-text">Please see below your userName and temporary password. When you click the login link, you will be re-directed to a screen where you will be asked to change your password.</mj-text>
+        <mj-text mj-class="body-text">Please see below your userName and temporary password. When you click the login link, you will be re-directed to a screen where you will be asked to change your temporary password.</mj-text>
```
Comment (plittlewood-rpt) [inline packages/cognito-custom-mail-lambda/src/mailer/templates/admin-user-invite.mjml:33]: Slightly against the ticket, but reading this doesn't read brilliantly as we are using the word temporary twice in the same sentence. Please can you just revert this change

**Review by plittlewood-rpt — CHANGES_REQUESTED** [review]
Please look at the build failure reasons. PRs need to have a title in a specific format in this project. Let me know if you can't figure it out (previous PRs will help you here too, it's just a regex)

File: packages/cognito-custom-mail-lambda/src/mailer/templates/admin-user-invite.mjml (hunk 2 above)
Comment (ashindey12) [reply packages/cognito-custom-mail-lambda/src/mailer/templates/admin-user-invite.mjml]: Updated the text
